content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -1,15 +1,13 @@
 ---
-title: DNS Proxying
-description: How to configure DNS proxying.
+title: DNS Proxying [Experimental]
```

Comment:
cc @ostromart Can you take a look at this page and tell me if it's labeled correctly.

Reply:
I don't see it listed in https://preliminary.istio.io/latest/about/feature-stages/

Reply:
I'm pretty sure this feature is supposed to be alpha in 1.9.

@howardjohn Please confirm.

Reply:
Yes, should be alpha